vscode-cairo/src/extension.ts

```diff
@@ -1,18 +1,21 @@
 import * as vscode from "vscode";
 import * as lc from "vscode-languageclient/node";
 import { setupLanguageServer } from "./cairols";
+import { Context } from "./context";
 
 let client: lc.LanguageClient | undefined;
 
-export async function activate(context: vscode.ExtensionContext) {
-  const config = vscode.workspace.getConfiguration();
-  const outputChannel = vscode.window.createOutputChannel("Cairo extension");
-  context.subscriptions.push(outputChannel);
+export async function activate(extensionContext: vscode.ExtensionContext) {
+  const ctx = Context.create(extensionContext);
```

Comment:
I wonder if that isn't something that we'd like to keep around globally for easier import and avoidance of prop-drilling